app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if (len(parts) != 3
+            or parts[0] != "cp"
+            or parts[1] == parts[2]):
+        return
+    with (open(parts[1], "r") as file_in,
+          open(parts[2], "w") as file_out):
+        file_in_read = "".join(file_in.read())
+        file_out.write(str(file_in_read))
```

Comment:
There is no need to create a `file_in_read` variable and cast string to string several times here. You could pass read directly to write, e.g.:
```suggestion
        file_out.write(file_in.read())
```
Rewrite the code to follow this approach.